src/test/resources/Features/OCD-2873-SimplifiedEditingOfDeveloper.feature

```diff
@@ -0,0 +1,64 @@
+@Regression
+Feature: OCD-2873 Simplified Editing of Developer
+  This test checks whether edit, split and merge developer functionality work as expected
+
+Scenario Outline: User logs in as ROLE and sees developer edit link exists on new developer page
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the developer edit page for developer "1854"
+    Then I see developer edit link for developer "1854"
+    Examples:
+      |ROLE       |
+      |ROLE_ACB   |
+      |ROLE_ADMIN |
+      |ROLE_ONC   |
+
+Scenario Outline: User logs in as ROLE and sees developer split link exists on new developer page
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the developer edit page for developer "1854"
+    Then I see developer split link for developer "1854"
+    Examples:
+      |ROLE       |
+      |ROLE_ACB   |
+      |ROLE_ADMIN |
+      |ROLE_ONC   | 
+
+Scenario Outline: User logs in as ROLE and sees developer merge link exists on new developer page
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the developer edit page for developer "1854"
+    Then I see developer merge link for developer "1854"
+    Examples:
+      |ROLE       |
+      |ROLE_ADMIN |
+      |ROLE_ONC   |
+
+Scenario Outline: User logs in as ROLE and edits a developer
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the developer edit page for developer "<developer id>"
+    And I edit for developer "<developer id>"
+    Then I see the edited developer information is recorded and updated on View Developer page
```

Comment:
I feel like it would be better to specify here in the feature what the developer name should be edited to and check against that value rather than using the getCurrentDate() method to generate a name.

Reply:
This is one of the paradigms we have for automated tests. We can't set the name to a specific value, because if we do that, and it works "today", when the tests run "tomorrow" we won't know if we actually change the name tomorrow or not, as it won't be a change. We do something similar in the "upload" tests, where the Product and Version Codes get set to values dependent on the date. I don't like the _format_ of the name that's being used, but it's functional

Reply:
Oh.. hmm.. when I was looking at ACB names a while ago it looked to me like in those tests we rename the ACB to a certain value and then later on in the test rename it back to something else. Also, larger issue, but I feel like we should always be starting a test suite from a known state of data - meaning we should run the bamboo job before the automated tests run. Otherwise how could we EVER assume that any acb, developer, whatever had a certain name? I could log in as an admin and rename the developer during the workday to be today's date and then you'd have the same problem you described above - how would you know if the edit worked?

Reply:
In the ACB rename one we'd done that all in the same step, including the asserts. That isn't great as it combines the action with the assert, but was the only way we could undo the things. And we needed to undo things on an ACB rename because otherwise all of the other tests using upload / filter / etc. failed.

With Developers, if a name changes here or there, unless we're using that Developer in another test it won't have any lasting impact. I'm up for ideas about undoing things in general, but we don't have that really set up right now. Maybe have jenkins do a database dump before running the tests, and then a restore after? Would definitely slow down the system, and might need to stop tomcat at the same time, which is complicated.

As for the "known state", I agree, but the hard part is that our users are messing with our state on a daily basis. For that reason I consider most of these tests as slightly fragile, but it's an issue we can't really do much about, outside of setting up an environment that doesn't have end user input